nvflare/private/fed/client/client_executor.py

```diff
@@ -214,6 +185,18 @@ def start_app(
         )
         thread.start()
 
+    def _get_job_launcher(self, job_id, client, job_meta: dict) -> dict:
```

Comment:
I thought we will simply fire an event here, and all launchers will listen to this event and decide whether they can handle this job.